flatbuffers/flatbuffers_case_study.md

```diff
@@ -0,0 +1,151 @@
+Case study of Javascript Flat Buffers
```

Comment:
You're right, but the post talks about our specific experience. What about "Refactoring JSON to FlatBuffers", "Moving from JSON to FlatBuffers", "Case study of JSON to FlatBuffers transition". Any ideas?